Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PolDiffILLReduction.py

```diff
@@ -655,11 +655,15 @@ def _normalise_vanadium(self, ws):
         return ws
 
     def _set_units(self, ws, process):
+        unit = 'd\sigma / d\Omega ({})'
```

Comment:
I meant removing the argument in parentheses. It is the x-axis unit that conveys this information, so keeping here in parentheses seems redundant. Instead, it should be the unit of the measure, e.g. barns/steradian/formula unit.

Reply:
Ok, I see you reverted this in another commit.